text-to-speech/General_TTS_inference_API_notebook_version2.ipynb

```diff
@@ -0,0 +1,196 @@
+        "id": "ow1mov-PhZmN"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "config_path=\"/content/vits_model/tts_models--en--ljspeech--vits/config.json\"\n",
+        "model_path=\"/content/coqui_vits.onnx\"\n",
+        "config = VitsConfig()\n",
+        "config.load_json(config_path)\n",
+        "vits = Vits.init_from_config(config)\n",
+        "vits.load_onnx(model_path)"
+      ],
+      "metadata": {
+        "id": "WAK5RnRToUYe"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "def tts_call(text:str):\n",
+        "  text_inputs = np.asarray(\n",
+        "      vits.tokenizer.text_to_ids(text, language=\"en\"),\n",
+        "      dtype=np.int64,\n",
+        "  )[None, :]\n",
+        "  audio = vits.inference_onnx(text_inputs)\n",
+        "  with tempfile.NamedTemporaryFile(suffix=\".wav\", delete=False) as temp_file:\n",
+        "    out_path = temp_file.name\n",
+        "  save_wav(wav=audio[0], path=out_path,sample_rate=22050)\n",
+        "  return out_path"
+      ],
+      "metadata": {
+        "id": "9uJ8N1JojC91"
+      },
+      "execution_count": 24,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "app = FastAPI()\n",
+        "app.add_middleware(\n",
+        "    CORSMiddleware,\n",
+        "    allow_origins=['*'],\n",
+        "    allow_credentials=True,\n",
+        "    allow_methods=['*'],\n",
+        "    allow_headers=['*'],\n",
+        ")"
+      ],
+      "metadata": {
+        "id": "XOQw6YSsNV5e"
+      },
+      "execution_count": 25,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "class TextRequest(BaseModel):\n",
+        "    text: str\n",
+        "\n",
+        "class OutputText(BaseModel):\n",
+        "  answer: str"
+      ],
+      "metadata": {
+        "id": "okc52MET8GZp"
+      },
+      "execution_count": 26,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "@app.get(\"/synthesize_audios\")\n",
+        "def synthesize_audio(payload: TextRequest):\n",
+        "    # Call your TTS function to generate the audio file\n",
+        "    audio_file_path = tts_call(payload.text)\n",
+        "    # Return the audio file as the response\n",
+        "    return FileResponse(audio_file_path, media_type=\"audio/wav\")\n",
+        "\n",
+        "@app.get(\"/\", response_model=OutputText)\n",
+        "async def root():\n",
+        "    return {\"answer\": \"Hello from TTS\"}"
+      ],
+      "metadata": {
+        "id": "xbHL2ZCNNYu3"
+      },
+      "execution_count": 27,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "#Replace comment section below with your auth token\n",
+        "!ngrok config add-authtoken #place your access token here"
```

Comment:
let's switch this to import the token from the secrets section. Similar to the change in https://github.com/brilla-ai/brilla-ai/commit/9bce29108eafea28da64bdc4b9d755cce7deee91#diff-e237dc95b672c52b333a45cb4f27cba37eecb11242ea598c779f0519b26439a6R24 and https://github.com/brilla-ai/brilla-ai/commit/9bce29108eafea28da64bdc4b9d755cce7deee91#diff-e237dc95b672c52b333a45cb4f27cba37eecb11242ea598c779f0519b26439a6R131